docs/clients/service-providers.md

```diff
@@ -18,17 +18,17 @@ There are three main pieces to consider:
 
 We will describe the steps to run and interract with a full-node for the Cosmos Hub. For other SDK-based blockchain, the process should be similar. 
 
-First, you need to [install the software](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/getting-started/installation.md).
+First, you need to [install the software](https://cosmos.network/docs/getting-started/installation.html).
```

Comment:
Shouldn't we be using relative links? https://github.com/cosmos/cosmos-sdk/blob/develop/docs/DOCS_README.md#links

Reply:
Yeah lets keep these as relative links and if anything have a search/replace before pushing it out to the website?  It's going to be a nightmare for us to maintain relative consistency otherwise

Reply:
Yeah isn't there a way to do ../ or something to make this work?
I think converting these to point to the website would be a nightmare to manage.

Reply:
Sure, will do